### ISSUE TITLE
tabBarBadge not working at bottomTabNavigator

### ISSUE THREAD

**Issue by ugrdursun** [react-navigation/react-navigation#8758]
**Current Behavior**

tabBarBadge not working on BottomTabNavigator as per documentation but its working perfectly on MaterialBottomTabNavigator.

**Expected Behavior**

tabBarBadge to work

**How to reproduce**

`                    <MainTab.Screen 
    
    name="Friends" 
    component={FriendsStackContainer}
    options={{
      
          tabBarBadge: 1,
          headerShown: true,
          tabBarLabel: 'Friends',
        }} />`

**Not working on bottomTabNavigator**, (same code works with materialBottomTabNavigator.)
Basically tabBarBadge prop not exists.

https://imgur.com/IzB7VCe

**Your Environment**

| software                      | version |
| ----------------------------- | ------- |
| iOS or Android                |
| @react-navigation/native     |5.0.9
| @react-navigation/bottom-tabs | 5.8.0
| react-native-screens          |
| react-native                  | 0.61.5
| expo                          | -
| node                          |
| npm or yarn                   |

**Comment by ugrdursun** [comment]
I cant understand why repo needed here ? I think problem is very clear. tabbarBadge just not working on bottomTabsNavigator.

My versions:

```
    "@react-navigation/bottom-tabs": "^5.8.0",
    "@react-navigation/material-bottom-tabs": "^5.2.16",
    "@react-navigation/material-top-tabs": "^5.1.4",
    "@react-navigation/native": "^5.0.9",
    "@react-navigation/stack": "^5.1.1",
    "react": "16.9.0",
    "react-native": "0.61.5",
```

**Comment by raajnadar** [comment]
You need to show the code , so that we can test it. Repro gives priority to fix the issue faster  we get many issues daily we can't create repro for each and test it.

If you provide a snack we can test it quickly and fix it.

**Comment by ugrdursun** [comment]
> The versions mentioned in the issue for the following packages differ from the latest versions on npm:
> 
> * `@react-navigation/native` (found: `5.0.9`, latest: `5.7.3`)
> 
> Can you verify that the issue still exists after upgrading to the latest versions of these packages?

Sorry for my bad, updating to latest version solved the problem. Can close the issue.
